tigergraph/ddl/load_dynamic.gsql

```diff
@@ -23,55 +23,159 @@ CREATE LOADING JOB load_dynamic FOR GRAPH ldbc_snb {
   DEFINE FILENAME file_Post_hasCreator_Person;
   DEFINE FILENAME file_Post_hasTag_Tag;
   DEFINE FILENAME file_Post_isLocatedIn_Country;
+  //load schema group 1
+  // load dynamic vertex
+  LOAD file_Comment 
+    TO VERTEX Comment1 VALUES ($1, ToMiliSeconds($0), $2, $3, $4, $5) USING header="true", separator="|";
```

Comment:
LOAD file_Comment 
    TO VERTEX Comment1 VALUES ($1, ToMiliSeconds($0), $2, $3, $4, $5) USING header="true", separator="|"
    TO VERTEX Comment2 VALUES ($1, ToMiliSeconds($0), $2, $3, $4, $5) USING header="true", separator="|"
    TO VERTEX Comment3 VALUES ($1, ToMiliSeconds($0), $2, $3, $4, $5) USING header="true", separator="|";